src/ducks/sharing/share.styl

```diff
@@ -0,0 +1,60 @@
+@import 'cozy-ui'
+
+:global
+    label[for='pho-album-share-toggle']
+        cursor pointer
+
+:local
+    .pho-share-modal-content
```

Comment:
That is why it is a bad idea to name thing with type or namespace when it is not specific...